internal/export/database/export.go

```diff
@@ -563,13 +563,14 @@ func (db *ExportDB) FinalizeBatch(ctx context.Context, eb *model.ExportBatch, fi
 		// Update ExportFile for the files created.
 		for i, file := range files {
 			ef := model.ExportFile{
-				BucketName: eb.BucketName,
-				Filename:   file,
-				BatchID:    eb.BatchID,
-				Region:     eb.Region,
-				BatchNum:   i + 1,
-				BatchSize:  batchSize,
-				Status:     model.ExportBatchComplete,
+				BucketName:   eb.BucketName,
+				Filename:     file,
+				BatchID:      eb.BatchID,
+				Region:       eb.Region,
```

Comment:
It might be worth renaming Region to OutputRegion to spare some confusion in the future.

This is particularly true now that it has no necessary relationship to regions as they are stored in Exposure table - this is basically just a string now that gets put in export files.

Reply:
OK if I do that rename in code, but not in database? 

trying to avoid unnecessary DB changes at this stage.

Reply:
Yeah, makes sense. Is it possibly easier to add a new column for it and no-op the current one (to eventually remove at some future time)? I've been spoiled working with tables of protos where field deprecation isn't too painful. :) 

Whichever works.

Reply:
updated.